zfsbootmenu/lib/kmsg-log-lib.sh

```diff
@@ -84,3 +84,49 @@ zerror() {
   : > "${BASE}/have_errors"
   echo "<3>ZFSBootMenu: $1" > /dev/kmsg
 }
+
+# arg1: comma-separated log levels to print
+# prints: all logs at that level
+# returns: nothing
+
+print_kmsg_logs() {
+  local levels levels_array grep_args
+
+  levels="${1}"
+  if [ -z "${levels}" ]; then
+    zerror "levels is undefined"
+    return
+  fi
+
+  # Try to use feature flags found in dmesg from util-linux
```

Comment:
There should probably be a better busybox vs util-linux dmesg detection routine ... somewhere.